industrial/start-container-kinetic.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.0"
+services:
```

Comment:
I would make a single docker-compose file for both kinetic and melodic, using different services.

Reply:
Done